### PR TITLE
Final

### PR DESCRIPTION
**Pull request by webmarkyn**

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Splendid work team :clap: . However, in order to get approval for this project, further changes needs to be made. check below :arrow_down: 

#### main content

- **[REQUIRED]** You footer width is causing an horizontal scrolling effect on you page. :pray: fix it
image

- **[REQUIRED]** Check the original page and try to add a similar bottom banner that is fixed at the bottom of the viewport to you page.
- **[REQUIRED]** Add more info to your readme file ( a short description of the project, technologies used , link to the assignment, link to your github profiles, etc...)

##### Bottom part
* [ ] It contains links to other articles with image previews on the leftmost side.
* [ ] It contains links to more popular articles on the rightmost side.

- **[:skull:]** I saw the list of commits and I noticed that you are not using the regular workflow (30min-switch). @rkyankya has done only the first commit. :worried: . Can you explain it guys ?

### That's it guys :relaxed: . When finished, kindly resubmit another code review form :pray: .

**Comment by webmarkyn** [comment]
@rkyankya get errors when trying to push to this project so he's giving me his code packed in zip and I push it.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,581 @@
...
+  margin: 0 auto;
+  padding: 5px;
+}
+
+/* Fixed banner */
+
+.fixed-banner {
+  position: fixed;
+  bottom: 0;
+  left: 0;
```
Comment (samgaco) [inline styles.css:567]: :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
-# micro-floating
\ No newline at end of file
+# Project: Position and floating with CSS3
+
+## Description
```
Comment (samgaco) [inline README.md:3]: You can still make the readme file look more professional, find inspiration [here](https://github.com/othneildrew/Best-README-Template) or [here](https://www.udacity.com/course/writing-readmes--ud777).

**Review by samgaco — APPROVED** [review]
Hi @webmarkyn  and @rkyankya :high_brightness: 

Good job, the project is approved :white_check_mark: you may merge to the master branch and submit the project completion form.

-----------------------------
* **About your issue,** try to find out what is happening and solve it so you both can commit in your next projects :pray: 
--------------------------------

Have this into account moving on :arrow_down: 
* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in your projects. For example now you are doing the pull request from a development branch, but it should be from a feature branch (some name that represents the feature you are working on). Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 
* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

----------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**